polaris/hub/client.py

```diff
@@ -866,12 +869,15 @@ def evaluate_competition(
              A `BenchmarkResults` object.
         """
         response = self._base_request_to_hub(
-            url="/v2/competition/evaluate",
+            url=f"/v2/competition/{competition.owner}/{competition.name}/evaluate",
             method="PUT",
-            json={"competition": competition.artifact_id, "predictions": y_pred},
+            json={"competition": competition.artifact_id, "predictions": y_pred, "access": access},
         )
-        return BenchmarkResults(
-            results=pd.read_json(response["scores"]),
-            benchmark_name=competition.name,
-            benchmark_owner=competition.owner,
+
+        # Inform the user about where to find their newly created artifact.
+        result_url = urljoin(
```

Comment:
Can we grab this URL from the response Location header instead of constructing it manually?

Reply:
We could, I was just following the pattern from the Benchmark evaluate function